installer/targets/tue-dev/install.yaml

```diff
@@ -49,26 +49,29 @@
 - type: target
   name: sergio-start
 
-# Handy shit
-- type: target
-  name: sublime3
-- type: target
-  name: terminator
-- type: target
-  name: vim
-- type: target
-  name: chrome
-- type: target
-  name: gitsu
-
 # Console
 - type: target
   name: ros-nl_robot_console
 
 # ED rviz plugin
 - type: target
   name: ros-ed_rviz_plugins
-  
+
 # HSR commands
 - type: target
   name: hsr-user
+
```

Comment:
Now you've just moved it. 

For the sake of compose-ability, make separate tue-dev and tue-tools targets, which are disjoint. For your matthijs target, you can include both but perhaps a ```tue-dev-full``` target also includes both tue-dev and tue-tools